android/src/main/java/com/swmansion/gesturehandler/react/RNGestureHandlerModule.kt

```diff
@@ -407,6 +407,10 @@ class RNGestureHandlerModule(reactContext: ReactApplicationContext?)
   fun handleClearJSResponder() {
   }
 
+  @ReactMethod
+  fun flushQueuedHandlers() {
+  }
```

Comment:
Would it be possible to define this method as no-op in pure JS, similarly as for web implementation?

Reply:
[Yes, it would](https://github.com/software-mansion/react-native-gesture-handler/pull/1979/commits/d583ce848e7ff49a22b1ee64de9e9b222f6b4912).